### PR TITLE
PA 2007

### PR DESCRIPTION
**Pull request by photon-rain**

### CODE REVIEW

File: meet-data/pa/0719/entries.csv [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Chris Catabiano,M,U/23,98.2,100,205,220,230,230,180,-190,-190,180,215,230,237,237,647,SBD,Single-ply,1
+Tristan Gassmen,M,O,98.3,100,170,180,-190,180,-150,155,165,165,-210,220,230,230,575,SBD,Single-ply,2
+Damien Giles,M,U/23,100.5,110,-230,230,250,250,170,181,-185,181,-270,-270,270,270,701,SBD,Single-ply,1
+Durand Welsh,M,O,109.1,110,215,-225,230,230,-160,160,170,170,215,-225,,215,615,SBD,Single-ply,1
+Jeff Tan,M,O,108.8,110,165,175,-185,175,115,125,-130,125,180,200,-210,200,500,SBD,Single-ply,2
+Steven Lewis,M,O,117.4,125,-180,180,200,200,120,-125,,120,225,265,-272.5,265,585,SBD,Single-ply,1
+Leon Pozinski,M,O,113.1,125,150,170,185,185,115,122.5,132.5,132.5,235,250,260,260,577.5,SBD,Single-ply,2
+Joe McIvor,M,O,112.6,125,-230,230,240,240,-145,145,-155,145,-210,-210,-210,,,SBD,Single-ply,DQ
+Steven Vaughan,M,O,147.1,125+,220,240,-260,240,-210,220,-230,220,-220,240,255,255,715,SBD,Single-ply,1
+Bruce Cole-Clark,M,O/60,72.5,75,,,,,-112.5,120,-122.5,120,,,,,120,SBD,Single-ply,1
```
Comment (sstangl) [inline meet-data/pa/0719/entries.csv]: Bruce through Strase were bench-only per the results.csv.

File: meet-data/pa/URLLIST [hunk 2]
```diff
@@ -9,7 +9,6 @@ https://powerliftingaustralia.com/wp-content/uploads/results/2007/NSW_Country_Op
 https://powerliftingaustralia.com/wp-content/uploads/results/2007/NSW_Sydney_Grading_PL_and_BP_Comp_-_12_Feb_2007.html
 https://powerliftingaustralia.com/wp-content/uploads/results/2007/Queensland_PL_and_BP_Open_-_21_Apr_2007.html
 https://powerliftingaustralia.com/wp-content/uploads/results/2007/SE_Challenge_and_Club_Open_-_1_Dec_2007.html
-https://powerliftingaustralia.com/wp-content/uploads/results/2006/2006_World_Championships_Results.html
```
Comment (sstangl) [inline meet-data/pa/URLLIST]: Please do not remove anything from the `URLLIST` -- instead, just make `URL` files and it'll be handled normally. The meets removed here seem unrelated to the meets added though -- a bunch of championships from 2006 through 2000. An error?

File: meet-data/pa/pa-parse-post [hunk 3]
```diff
@@ -24,6 +24,8 @@ sed -i -e "s/,\(-[0-9][0-9]*\)\.0,/,\1,/g" entries.csv
 # Copy the template to the current directory for filling in.
 cp ${SCRIPTDIR}/../meet.template meet.csv
 
+echo PA >> meet.csv
```
Comment (sstangl) [inline meet-data/pa/pa-parse-post]: Could do one better and include a comma!

File: meet-data/pa/URLLIST (hunk 2 above)
Comment (photon-rain) [reply meet-data/pa/URLLIST]: Definitely error, just mean to remove the world champs & Oceania results.
